Jenkinsfile

```diff
@@ -38,17 +36,19 @@ pipeline {
             sh 'git tag -a $BRANCH_NAME-build-$BUILD_NUMBER -m "Jenkins"'
             sh 'git push https://${GIT_USR}:${GIT_PSW}@github.com/yonadev/yona-app-ios.git --tags'
           }
-          
         }
-        
         archiveArtifacts 'Yona/BuildOutput/**/*.ipa'
       }
       post {
         always {
           junit 'Yona/BuildOutput/**/*.xml'
-          
         }
-        
+        success {
+          slackSend color: 'good', channel: '#dev', message: "iOS app build ${env.BUILD_NUMBER} on branch ${BRANCH_NAME} succeeded"
+        }
+        failure {
+          slackSend color: 'bad', channel: '#dev', message: "iOS app build ${env.BUILD_NUMBER} on branch ${BRANCH_NAME} failed"
```

Comment:
I want to know difference between `${BUILD_NUMBER}` and `${env.BUILD_NUMBER}`.

Reply:
See above